### DISCUSSION TITLE
Add support for image width/height for Cloudinary images

### DISCUSSION THREAD

**Discussion by iljapanic** [keystonejs/keystone#8603] [Feature requests]
Currently, the GraphQL only returns the following fields when using @keystone-6/cloudinary field type:

```
filename
id
publicUrl
publicUrlTransformed
originalFilename
mimetype
```

It would be great to also get access to data about image width and height, so it's easier to integrate with Next.js `<Image />` component and others.

**Comment by ancashoria** [comment]
This is a big pain point for us.

**Comment by GhostOrder28** [comment]
I needed this feature in a project I'm currently working on, so looking at the codebase (keystone 6) I figured out you just need to add the some lines in the [cloudinary image field](https://github.com/keystonejs/keystone/blob/main/packages/cloudinary/src/index.ts), so just make a copy of it (and in my case I also made a copy for the frontend files) and add the following lines:

```
type CloudinaryImage_File = {
  ...
  width: number <----
  height: number <----
}
```

```
export const outputType: graphql.ObjectType<CloudinaryImage_File> =
  graphql.object<CloudinaryImage_File>()({
    name: 'CloudinaryImage_File',
    fields: {
      ...
      width: graphql.field({ type: graphql.Int }), <----
      height: graphql.field({ type: graphql.Int }), <----
    })
```

```
async function resolveInput () {
    return {
      ...
      width: _meta.width, <----
      height: _meta.height, <----
    }
}
```

Hope it helps.

**Reply by dcousens** [reply]
@GhostOrder28 did you want to make a pull request for this?
